src/SignalR/clients/ts/FunctionalTests/Startup.cs

```diff
@@ -36,12 +36,12 @@ public void ConfigureServices(IServiceCollection services)
             {
                 options.EnableDetailedErrors = true;
             })
-            .AddNewtonsoftJsonProtocol(options =>
-            {
-                // we are running the same tests with JSON and MsgPack protocols and having
-                // consistent casing makes it cleaner to verify results
-                options.PayloadSerializerSettings.ContractResolver = new DefaultContractResolver();
-            })
+            // .AddNewtonsoftJsonProtocol(options =>
+            // {
+            //     // we are running the same tests with JSON and MsgPack protocols and having
+            //     // consistent casing makes it cleaner to verify results
+            //     options.PayloadSerializerSettings.ContractResolver = new DefaultContractResolver();
+            // })
```

Comment:
Delete rather than comment out. I'm going to stop saying this though there's more cases below.